Modules/Filtering/MathematicalMorphology/include/itkMathematicalMorphologyEnums.h

```diff
@@ -0,0 +1,56 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkMathematicalMorphologyEnums_h
+#define itkMathematicalMorphologyEnums_h
+
+#include <iostream>
+#include "itkMathematicalMorphologyExport.h"
```

Comment:
This should be `#include "ITKMathematicalMorphologyExport.h"` (`ITK` uppercase.). But there is no compile attempt of corresponding .cxx file. No wonder that `ITKMathematicalMorphology.lib` does not exist. But I don't know why.

Reply:
Nice catch ! Addressed in be5455a.